services/QuillLMS/config/initializers/posthog_integration.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+module PosthogIntegration
+  API_KEY = ENV.fetch('PUBLIC_POSTHOG_KEY', '')
+  HOST    = ENV.fetch('PUBLIC_POSTHOG_HOST', 'https://us.i.posthog.com')
```

Comment:
I feel like this is a solid pattern for extracting ENV variables into a module scope.